forloop_modules/queries/node_context_requests_backend.py

```diff
@@ -1072,7 +1072,7 @@ def get_all_databases_by_project_uid():
     if response.status_code != 200:
         raise Exception(f'Error {response.status_code}: {response.reason}.')
     
-    databases = response.json().get("result", {}).get("databases")
+    databases = response.json()['databases']
```

Comment:
maybe .get("databases") still better?

Reply:
`databases` attr should always be sent in the response - using `get()` if it does not exist will silence the exception, propagating it somewhere down